typescript/packages/wallets/crossmint/src/custodial.ts

```diff
@@ -84,9 +85,10 @@ export function custodialFactory(apiKey: string) {
             },
             async sendTransaction({ instructions }: SolanaTransaction) {
                 const latestBlockhash = await connection.getLatestBlockhash("confirmed");
+                // see https://linear.app/crossmint/issue/WAL-3436/solana-transactions-get-stuck-when-sending-the-dummy-value
```

Comment:
LEave crossmint references outside of this public repo. If it's relevant to goat pls file a public github issue instead

Reply:
Ok, its in the project I mentioned here https://crossmint.slack.com/archives/C07TF9ZRNG7/p1733502896139059